videoroom/5_ImplementingServerRoom.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 5. Server's room
-description: >- 
-  Create your very own videoconference room with a little help from the Membrane Framework!
+description: >-
+  Create your very own video conference room with a little help from the Membrane Framework!
```

Comment:
In case we wanted to change it here, we would need to change the spelling everywhere. I can push a commit with that changes, however I am not sure if writing it as "videoconference" is a mistake - Oxford Dictionary has a ["videoconferencing" record](https://www.oxfordlearnersdictionaries.com/definition/english/videoconferencing?q=videoconferencing)

Reply:
I've just accepted a suggestion from spell-checking inside VSCode. As you've noticed, this could be `a videoconferencing room` and I think it sounds better, so I've changed that in all occurences